Makefile

```diff
@@ -1,13 +1,18 @@
-sources = isrgrootx1.signing_policy.txt \
-          letsencryptauthorityx3.signing_policy.txt \
-          letsencryptauthorityx4.signing_policy.txt
+sources = isrgrootx1.signing_policy \
+          letsencryptauthorityx3.signing_policy \
+          letsencryptauthorityx4.signing_policy \
+	  lets-encrypt-r3.signing_policy \
+	  lets-encrypt-r4.signing_policy \
```

Comment:
Whitespace

```suggestion
          letsencryptauthorityx3.signing_policy \
          letsencryptauthorityx4.signing_policy \
          lets-encrypt-r3.signing_policy \
          lets-encrypt-r4.signing_policy \
```

Reply:
I saw the weirdness in the diff yesterday and was a little confused because the lines visually lined up in my editor/on GitHub. It looks like my editor's adding tabs instead of spaces in Makefiles but spaces everywhere else!